web/source/dom/preProcessor.ts

```diff
@@ -193,6 +193,15 @@ namespace com.keyman.dom {
           };
         }
       }
+
+      // Check for any browser-based keymapping before returning the object.
+      if(!keyman.isEmbedded && s.device.browser == text.Browser.Firefox) {
```

Comment:
Why the `isEmbedded` check? I would think that (a) it could never be true for Firefox, and (b) even if it was, this transform would still be necessary?

Reply:
... yeah, I kind of had that thought, too.  But, this kept it true to the source line that was moved.  We can totally drop the `isEmbedded` check.